pkg/manager/manager.go

```diff
@@ -12,11 +12,16 @@ import (
 	"github.com/GreptimeTeam/gtctl/pkg/helm"
 	"github.com/GreptimeTeam/gtctl/pkg/kube"
 	"github.com/GreptimeTeam/gtctl/pkg/log"
-	"github.com/GreptimeTeam/gtctl/pkg/utils"
 )
 
 const (
-	defaultChartsURL = "https://github.com/GreptimeTeam/helm-charts/releases/download"
+	defaultChartsURL                      = "https://github.com/GreptimeTeam/helm-charts/releases/download"
+	DefaultGreptimeDBChartVersion         = "0.1.0"
```

Comment:
I think we should always use the latest version if the user don't specify the version. It should not hard code the version.

Reply:
I will set in the next pr, pull chart latest version